src/SDKs/Batch/DataPlane/Azure.Batch/Common/Enumerations/TaskCountValidationStatus.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Batch.Common
+{
+    /// <summary>
+    /// Whether the task counts have been validated.
+    /// </summary>
+    public enum TaskCountValidationStatus
+    {
+        /// <summary>
+        /// The task counts have been validated and are guaranteed to match the List Tasks results.
```

Comment:
'guaranteed' may be strong given the non-atomicity of List Tasks, plus I assume there must be *some* latency